airbyte-integrations/connectors/destination-iceberg/src/main/resources/spec.json

```diff
@@ -154,6 +154,57 @@
                 "order": 3
               }
             }
+          },
+          {
+            "title": "NessieCatalog",
+            "description": "The NessieCatalog connects to a Nessie server at the specified URI",
+            "required": ["catalog_type", "nessie_uri"],
+            "properties": {
+              "catalog_type": {
+                "title": "Catalog Type",
+                "type": "string",
+                "default": "Nessie",
+                "enum": ["Nessie"],
+                "order": 0
+              },
+              "nessie_uri": {
+                "title": "Nessie Server URI",
+                "type": "string",
+                "default": "http://localhost:19120/api/v1",
```

Comment:
Please default to v2 (v1 is deprecated)

Reply:
Sorry, I just realised that `NessieCatalog` in Iceberg 1.3.0 (current dependency version in Airbyte) only supports API v1 :facepalm:

Reply:
Can we have a PR to bump the iceberg version first? So that we don't have to keep V1 examples?